rocks-db/src/columns/offchain_data.rs

```diff
@@ -0,0 +1,138 @@
+use crate::column::TypedColumn;
+use crate::generated::offchain_data_generated::off_chain_data as fb;
+use crate::key_encoders::{decode_string, encode_string};
+use crate::{Result, ToFlatbuffersConverter};
+use entities::models::OffChainDataGrpc;
+use flatbuffers::{FlatBufferBuilder, WIPOffset};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub enum StorageMutability {
+    #[default]
+    Immutable,
+    Mutable,
+}
+
+impl StorageMutability {
+    pub fn is_mutable(&self) -> bool {
+        match self {
+            StorageMutability::Immutable => false,
+            StorageMutability::Mutable => true,
+        }
+    }
+}
+
+impl From<&str> for StorageMutability {
+    fn from(storage_mutability: &str) -> Self {
```

Comment:
What is storage_mutability in this case? Is it the uri? with ipfs it has the ipfs as protocol at times, like ipfs://<hash>, while arweave is working over https and may have www prefix as well
Some examples:
ipfs://bafybeib3ogmisx45y45er4jjxi3nd7cta5smeapdoueodkt363g4loeisa/36515.json
"https://www.arweave.net/Hkldep0np3sNm8HMj_e7OwoRUy0XVqD8FKYchcxLoUs?ext=json"
"https://arweave.net/kSb4SqqpGwdVVeyQ8JPhT_htIbq6Mn0gWLguPqF5gow"
"https://www.arweave.net/_n-rDupBSHsJoA9qDux-eR7KjLxaXR5TE7724EAazcw?ext=json"

Reply:
Thanks!